proto/Makefile

```diff
@@ -33,3 +33,12 @@ build: $(patsubst %.proto, %.pb.go, $(shell find src/determined -type f -name '*
 .PHONY: check
 check:
 	buf check lint
+
+.PHONY: check-fmt
+check-fmt:
```

Comment:
I'll never remember this target exists. Can it just run as part of `check` above?

Reply:
Oh wait no, because then CI would fail 🤦

Reply:
Really unfortunate.

Reply:
yeah but only until we add it to CI.